libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardView/DashboardView.tsx

```diff
@@ -100,6 +107,20 @@ export const DashboardView: React.FC<IDashboardViewProps> = ({
         <DashboardViewConfigProvider config={effectiveConfig}>
             <UserWorkspaceSettingsProvider settings={userWorkspaceSettings}>
                 <ColorPaletteProvider palette={colorPalette}>
+                    {isScheduledMailDialogVisible && (
+                        <ScheduledMailDialog
+                            backend={backend}
+                            workspace={workspace}
+                            locale={config?.locale}
```

Comment:
The locale is more involved than this, please see DashboardView component (we must default to user settings if config does not specify locale).

Reply:
This is actually solved on ScheduledMailDialog component level, but we should probably default it also here so it's consistent.